DOC-7888 Collection Manager

osfameron opened the pull request.

osfameron committed:
DOC-7888 Collection Manager

maria-robobug commented on modules/howtos/examples/collection-manager.php:
```diff
@@ -0,0 +1,101 @@
+    // end::create-scope[]
+
+    print "scopeAdmin\n";
+	// tag::scopeAdmin[]
```
got an extra space here I think?

osfameron replied:
woo, so what's happened is that Go uses tabs, and I copied the examples from the Go project while translating them. So any thing I typed new was with spaces, but the tags were left unchanged, so stayed as tabs...

Didn't show up on my editor of course... will fix now, thanks.

maria-robobug commented on modules/howtos/examples/collection-manager.php:
```diff
@@ -0,0 +1,101 @@
+    ]);
+
+    $users->upsertUser($user);
+	// end::scopeAdmin[]
```
same here, extra space

maria-robobug commented on modules/howtos/examples/collection-manager.php:
```diff
@@ -0,0 +1,101 @@
+function getCollections($username, $password) {
+    print "create-collection-manager\n";
+
+	// tag::create-collection-manager[]
```
extra space here

maria-robobug commented on modules/howtos/examples/collection-manager.php:
```diff
@@ -0,0 +1,101 @@
+    $bucket = $cluster->bucket("travel-sample");
+
+    $collections = $bucket->collections();
+	// end::create-collection-manager[]
```
same here

maria-robobug commented on modules/howtos/examples/collection-manager.php:
```diff
@@ -0,0 +1,101 @@
+    print "create-collection-manager\n";
+
+	// tag::create-collection-manager[]
+    $options = new \Couchbase\ClusterOptions();
```
Do we need to re-use the fully qualified name here if we've actually imported above with `use`?
I'm never sure about this with PHP 😕

osfameron replied:
I think can omit, but may be clearer to keep it, and remove the explicit import (which I copied from a previous example...)

maria-robobug reviewed (approved):
LGTM, just had one question about the imports 👍

osfameron committed:
DOC-7888 whitespace fixes

fixing tabs

(introduced by workflow of translating these examples from Go, where Tabs are
the standard indentation)

osfameron committed:
DOC-7888 omit imports

As the namespaces are all fully qualified in the sample code,
the imports are not needed.

osfameron commented:
Merging as review issues addressed with no further comments.
